torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -868,12 +877,38 @@ def escape_string(x):
                     .replace("\t", "\\t")
                 )
 
-            self.prefix.writeline(
-                f'in_spec_ = R"({config.aot_inductor.serialized_in_spec})";'
-            )
-            self.prefix.writeline(
-                f'out_spec_ = R"({config.aot_inductor.serialized_out_spec})";'
-            )
+            if (
+                len(config.aot_inductor.serialized_in_spec)
+                > MSVC_C2026_MAX_STRING_LENGTH
+            ):
+                serialized_in_spec_strs = truncate_string(
+                    config.aot_inductor.serialized_in_spec, MSVC_C2026_MAX_STRING_LENGTH
+                )
+                self.prefix.writeline("in_spec_ =")
+                for truncate_str in serialized_in_spec_strs:
+                    self.prefix.writeline(f'R"({truncate_str})"')
+                self.prefix.writeline(";")
+            else:
+                self.prefix.writeline(
+                    f'in_spec_ = R"({config.aot_inductor.serialized_in_spec})";'
+                )
+
+            if (
+                len(config.aot_inductor.serialized_out_spec)
+                > MSVC_C2026_MAX_STRING_LENGTH
+            ):
+                serialized_out_spec_strs = truncate_string(
+                    config.aot_inductor.serialized_out_spec,
+                    MSVC_C2026_MAX_STRING_LENGTH,
+                )
+                self.prefix.writeline("out_spec_ =")
+                for truncate_str in serialized_out_spec_strs:
+                    self.prefix.writeline(f'R"({truncate_str})"')
+                self.prefix.writeline(";")
+            else:
+                self.prefix.writeline(
+                    f'out_spec_ = R"({config.aot_inductor.serialized_out_spec})";'
+                )
```

Comment:
Put this duplicated pattern in a helper method (merge with the existing one).

Something like `self.write_multiline_string("out_spec_", config.aot_inductor.serialized_out_spec)`

Reply:
@jansel updated code as your comment.